msticpy/datamodel/entities/host_logon_session.py

```diff
@@ -73,7 +75,7 @@ def __init__(
         self.StartTimeUtc: datetime = datetime.min
         self.EndTimeUtc: datetime = datetime.min
         self.Host: Optional[Host] = None
-        self.SessionId: str = ""
+        self.SessionId: str | None = ""
```

Comment:
Since None is a valid type, why not initializing with None instead of an empty string?

Reply:
Hmm - ok. I think this change was a Ruff autofix. Not sure why it picked out this entity file. 
I can fix but don't ask me to update all the other stuff n the entities folder :-)